### ISSUE TITLE
Implement About dialog with Inputstream/Widevine details

### ISSUE THREAD

**Issue by dagwieers** [emilsvennesson/script.module.inputstreamhelper#87]
Currently on ARM we have no clue what version of Widevine CDM version we get from the ChromeOS image. While we can do it like this:
```shell
kodi01:~ # grep -E '[0-9]+\.[0-9]+\.[0-9]+\.[0-9]+' ~/.kodi/cdm/libwidevinecdm.so 
4.10.1440.18
4.10.1440.18
```

It would be nice if we can do this from python instead, preferably by reading the ELF structure.

**Comment by horstle** [comment]
I couldn't find the version string in the output of `readelf`, but this would do the same as your command:
```python
import re

def get_lib_version(file):
    with open(file, 'rb') as f:
        x = re.search('[0-9]+\.[0-9]+\.[0-9]+\.[0-9]+', str(f.read()))
        return(x.group(0).lstrip('0'))
```
With `file` being the path to the libwidevinecdm.so.

**Comment by dagwieers** [comment]
This would indeed be the alternative implementation (but is also a liability if it would fail).
We need unit-tests for this so we can detect issues and escalate them.

**Comment by dagwieers** [comment]
I verified and this works for Linux, Windows and macOS.

**Comment by dagwieers** [comment]
I think this is sufficient for showing the version in an About dialog in the settings.
Then that dialog could show:
- Version of InputStream Helper
- Current Widevine version
- Latest ChromeOS version used (stored in settings)
- Detected platform/architecture
- Location of the Widevine CDM library
- A link to where to report issues on GitHub

**Comment by dagwieers** [comment]
This is implemented with #139
